tasks/benchmark/benches/transformer.rs

```diff
@@ -39,6 +39,7 @@ fn bench_transformer(criterion: &mut Criterion) {
                 // `enable_all` enables all transforms except arrow functions transform
```

Comment:
```suggestion
                // `enable_all` enables all transforms except arrow functions transform
                // and async-to-generator
```

Reply:
The comment sounds incorrect, shouldn't include `arrow-functions` and `async-to-generator`?

Reply:
Now I understand